environment.yml

```diff
@@ -85,9 +85,8 @@ dependencies:
   # code checks
   - black=22.3.0
   - cpplint
-  - flake8=4.0.1
-  - flake8-bugbear=21.3.2  # used by flake8, find likely bugs
-  - flake8-comprehensions=3.7.0  # used by flake8, linting of unnecessary comprehensions
```

Comment:
Was this supposed to be removed?

Reply:
Yes, there's already pyupgrade for rewriting comprehensions